aclk/README.md

```diff
@@ -13,36 +13,96 @@ The Agent-Cloud link (ACLK) is the mechanism responsible for connecting a Netdat
 
-Read our [claiming documentation](/claim/README.md) for a guide for claiming a node using the ACLK and additional
-troubleshooting and reference information.
+For a guide for claiming a node using the ACLK, plus additional troubleshooting and reference information, read our
+[claiming documentation](/claim/README.md) or the [get started with
+Cloud](https://learn.netdata.cloud/docs/cloud/get-started) guide.
 
 ## Enable and configure the ACLK
 
 The ACLK is enabled by default and automatically configured if the prerequisites installed correctly. You can see this
-in the `[cloud]` section of `netdata.conf`.
+in the `[global]` section of `var/lib/netdata/cloud.d/cloud.conf`.
 
 ```conf
-[cloud]
-    cloud base url = https://app.netdata.cloud
+[global]
+  enabled = yes
+  cloud base url = https://app.netdata.cloud
 ```
 
 If your Agent needs to use a proxy to access the internet, you must [set up a proxy for
 claiming](/claim/README.md#claim-through-a-proxy).
 
 ## Disable the ACLK
 
-You have two options if you prefer to disable the ACLK and not use Netdata Cloud:
+You have two options if you prefer to disable the ACLK and not use Netdata Cloud.
+
+### Disable at installation
+
+You can pass the `--disable-cloud` parameter to the Agent installation when using a kickstart script
+([kickstart.sh](/packaging/installer/methods/kickstart.md) or
+[kickstart-static64.sh](/packaging/installer/methods/kickstart-64.md)), or a [manual installation from
+Git](/packaging/installer/methods/manual.md).
 
-1.  Pass `--disable-cloud` to `netdata-installer.sh` during installation. When you pass this parameter, the installer
-    does not download or compile any extra libraries, and the Agent behaves as though the ACLK, and thus Netdata Cloud,
-    does not exist. ACLK functionality is available in the Agent but remains fully inactive.
+When you pass this parameter, the installer does not download or compile any extra libraries, and the Agent behaves as
+though the ACLK, and thus Netdata Cloud, does not exist. ACLK functionality is available in the Agent but remains fully
+inactive.
 
-2.  Change a runtime setting in your `netdata.conf` file. This setting only stops the Agent from attempting any
-    connection via the ACLK, but does not prevent the installer from downloading and compiling the ACLK's dependencies.
+### Disable at runtime
+
+You can change a runtime setting in your `var/lib/netdata/cloud.d/cloud.conf` file to disable the ACLK. This setting
+only stops the Agent from attempting any connection via the ACLK, but does not prevent the installer from downloading
+and compiling the ACLK's dependencies.
+
+Change the `enabled` setting to `no`:
 
 ```conf
 [global]
-    netdata cloud = disable
+    enabled = no
+```
+
+If the file at `var/lib/netdata/cloud.d/cloud.conf` doesn't exist, you need to create it.
+
+```bash
+cd /var/lib/netdata/cloud.d
+cat > cloud.conf << EOF
```

Comment:
This is missing the piece of code below, they cannot be split like this as the here-doc needs to be contiguous:
```bash
cd /var/lib/netdata/cloud.d
cat > cloud.conf << EOF
[global]
    enabled = no
    cloud base url = https://app.netdata.cloud
EOF
```

Reply:
Maybe change the line to explain the change in prompt and leave it after the code?

> After you cut and paste the first two lines the prompt will change to `cat`, keep pasting in the lines and after the `EOF` your normal terminal prompt will come back. To get your prompt back the final line must contain only `EOF`, nothing else.